main.tf

```diff
@@ -1,13 +1,10 @@
-resource "juju_offer" "oauth_offer" {
-  name             = "oauth-offer"
-  model            = var.model
-  application_name = juju_application.hydra.name
-  endpoints        = ["oauth"]
+data "juju_model" "this" {
```

Comment:
this seems to break the tutorial:
```
│ Error: Client Error
│ 
│   with module.iam.data.juju_model.this,
│   on ../../main.tf line 1, in data "juju_model" "this":
│    1: data "juju_model" "this" {
│ 
│ Unable to read model, got error: model "iam" not found
```

Reply:
mind having another go at it?

Reply:
if it doesn't work we should simply add

```
depends_on = [ juju_model.iam ]
```

on line 83